spec/rails_helper.rb

```diff
@@ -435,59 +457,43 @@ def synchronize(seconds = nil, errors: nil)
       end
     end
 
-    # possible values: OFF, SEVERE, WARNING, INFO, DEBUG, ALL
-    browser_log_level = ENV["SELENIUM_BROWSER_LOG_LEVEL"] || "WARNING"
-
-    chrome_browser_options =
-      Selenium::WebDriver::Chrome::Options
-        .new(logging_prefs: { "browser" => browser_log_level, "driver" => "ALL" })
-        .tap do |options|
-          apply_base_chrome_options(options)
-          options.add_argument("--disable-search-engine-choice-screen")
-          options.add_argument("--window-size=1400,1400")
-          options.add_preference("download.default_directory", Downloads::FOLDER)
-        end
-
-    driver_options = { browser: :chrome, timeout: BROWSER_READ_TIMEOUT }
-
-    if ENV["CAPYBARA_REMOTE_DRIVER_URL"].present?
-      driver_options[:browser] = :remote
-      driver_options[:url] = ENV["CAPYBARA_REMOTE_DRIVER_URL"]
```

Comment:
We need to continue to support this as this is used by our `discourse_theme` CLI tool: https://github.com/discourse/discourse_theme/blob/cdc18da0174c2b70989424d5546749063698aed8/lib/discourse_theme/cli_commands/rspec.rb#L189